docs/querying/caching.md

```diff
@@ -82,3 +82,12 @@ Note that the task executor processes only support caches that keep their data l
 This restriction exists because the cache stores results at the level of intermediate partial segments generated by the
 ingestion tasks. These intermediate partial segments will not necessarily be identical across task replicas, so
 remote cache types such as `memcached` will be ignored by task executor processes.
+
+## Unsupported queries
+
+Query caching is not available for following
+- queries, that have a union operation, do not support result-level caching - [More details](https://github.com/apache/druid/issues/8713)
+- queries, that involve an Inline data source or a Lookup data source, do not support any caching. 
+- queries, with a sub-query in them, do not support any caching though the output of sub-queries itself may be cached.
```

Comment:
It would be worth linking https://druid.apache.org/docs/0.19.0/querying/query-execution.html#query to help understanding since this requires to know how subquery system works in Druid.

Reply:
Ack